test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- As this is clearly stated that `I should be ignored by git`, kindly add this file in the `.gitignore` file.